projects/mysadna/src/app/app.component.scss

```diff
@@ -1,56 +1,40 @@
-// portrait phones: 576px
-// landscape phones: 768px
-// tablets: 992px
-// desktops: 1200px
-
-.app-container {
-  font-family: 'OpenSans', sans-serif;
-  @media(max-width: 575.98px) {
-    font-size: 10px;
+$logo-size: 74px;
+.app-logo {
+  position: absolute;
+  top: 5px;
+  left: 5px;
+  justify-self: flex-start;
+  display: flex;
+  >.my-sadna-logo {
+    height: $logo-size;
+    width: $logo-size;
+  }
+  >.my-sadna {
+    margin-left: 10px;
+    font-size: 22px;
+    font-weight: 800;
+    color: #0097e1;
+    padding-top: 15px;
   }
 }
-
 .app-header {
   display: flex;
   justify-content: space-between;
-  @media (max-width: 575.98px) {
-    flex-direction: column;
-  }
-  >.app-logo {
-    justify-self: flex-start;
-    display: flex;
-    justify-content: space-around;
-    @media(max-width:575.98px) {
-      justify-content: flex-start;
-    }
-    >.my-sadna-logo {
-      height: 5em;
-      width: 5em;
-    }
-    >.my-sadna {
-      margin: 1em;
-      font-size: 1.50em;
-      font-weight: 800;
-      color: #0097e1;
-    }
+  padding: 5px;
+  >.placeholder {
+    height: $logo-size;
   }
   >.app-navigation-bar {
     justify-self: flex-end;
-    @media(max-width: 575.98px) {
-      align-self: center;  
-    }
+    padding: 18px;
     >.app-navigation {
       >a {
-        margin: 1em;
-        font-size: 1.25em;
+        margin: 20px;
+        font-size: 18px;
```

Comment:
I prefer to avoid relative font sizes, especially if we have mock in px

Reply:
Understood.
How do you know that the sizes in the mock won't differ between two different screens?

Reply:
@a3l5o7n9 It will differ and it's good.
I mean if font is 15px then it's going to be 15px on all screens, but it may look different, depends on screen resolution. Sometimes it's ok and it's the planed. Sometimes not, and if need we can font size can be dependent on device screen. But still we don't need to use em anyway, em is just more easiest way in some cases.